src/components/common/animations/Error/data.json

```diff
@@ -616,6 +616,218 @@
       "op": 92,
       "st": -4,
       "bm": 0
+    },
```

Comment:
I suppose that this adds the 'Ooops' text, It has been removed because in iOS it's not showed and in Android generate crashes, remove this pls.